<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Add Publication</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="5">
         <widget class="QPushButton" name="buttSave">
          <property name="text">
           <string>Hinzufügen</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="issue"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="butAddSeriesD">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/res/icon-plus.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item row="6" column="5">
         <widget class="QPushButton" name="butAddReprint">
          <property name="text">
           <string>Inhalt übernehmen</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Nummer (orig.)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="butAddSeriesEN">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/res/icon-plus.png</normaloff>:/res/icon-plus.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Nummer</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="cbSeriesOrig"/>
        </item>
        <item row="6" column="4">
         <widget class="QLineEdit" name="isuuePart"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Serie (orig.)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Serie</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Teil</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="pubNo">
          <property name="inputMask">
           <string comment="AAA999999"/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="butRemoveLine">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/res/icon-purge.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cbSeriesD"/>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="butBarcode">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/res/barcode.png</normaloff>:/res/barcode.png</iconset>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="issueEN"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Inhalt</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>PUB</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QListWidget" name="listWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_db">
       <attribute name="title">
        <string>Database</string>
       </attribute>
       <widget class="QTreeView" name="treeViewDB">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>661</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>pubNo</tabstop>
  <tabstop>cbSeriesD</tabstop>
  <tabstop>issue</tabstop>
  <tabstop>butRemoveLine</tabstop>
  <tabstop>cbSeriesOrig</tabstop>
  <tabstop>issueEN</tabstop>
  <tabstop>isuuePart</tabstop>
  <tabstop>butAddReprint</tabstop>
  <tabstop>buttSave</tabstop>
  <tabstop>butAddSeriesD</tabstop>
  <tabstop>butAddSeriesEN</tabstop>
  <tabstop>butBarcode</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pubNo</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>checkPub()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>issueEN</sender>
   <signal>returnPressed()</signal>
   <receiver>butAddReprint</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>isuuePart</sender>
   <signal>returnPressed()</signal>
   <receiver>butAddReprint</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkPub()</slot>
  <slot>refreshSeriesD()</slot>
  <slot>refreshSeriesEN()</slot>
 </slots>
</ui>
